src/components/DashboardCard/DashboardCard.scss

```diff
@@ -0,0 +1,79 @@
+@import "~sass-rem";
+@import "../../assets/mixin.scss";
+@import "../../assets/variable.scss";
+
+.dashboard-card {
+  width: rem(160px);
+  padding: rem(10px) rem(40px) rem(10px) rem(20px);
+  background: #ffffff;
+  box-shadow: 0px 11px 15px rgba(0, 0, 0, 0.02);
+  border-radius: rem(4px);
+  margin-bottom: rem(30px);
+    h1 {
+        font-family: Work Sans;;
```

Comment:
- you can remove the font family and it will still work fine

- you can use the $white variable instead of #ffffff

Reply:
Thanks... Done